data-plane/pom.xml

```diff
@@ -577,6 +589,11 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.antlr</groupId>
+        <artifactId>antlr4-maven-plugin</artifactId>
+        <version>${antlr.version}</version><!-- keep it aligned with the runtime bom-->
```

Comment:
The comment is not really telling what "runtime" we're referring to but also I'd place in on the version properties more than here